api/kyverno/v1alpha2/common_types.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2020 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	kyvernov1 "github.com/kyverno/kyverno/api/kyverno/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CleanupPolicySpec struct {
```

Comment:
A cleanup policy can only take one rule?

Reply:
@realshuting Could you please elaborate.

Reply:
I would keep it simple for now, extending to multiple rules should be relatively easy.

Reply:
@NikhilSharmaWe can you move it to v1alpha1 ? I can do it if it saves time.